From bc8e9890de8c9866e6e1f49c8355ebee915579e4 Mon Sep 17 00:00:00 2001
From: q66 <daniel@octaforge.org>
Date: Wed, 29 Jan 2020 12:50:46 +0100
Subject: [PATCH 1/2] always use IEEE floats

We don't build on any platforms where this is not the case.

Fixes platforms for which this is not checked.
---
 libsrc/Dk/Dkmarshal.c | 20 +-------------------
 1 file changed, 1 insertion(+), 19 deletions(-)

diff --git libsrc/Dk/Dkmarshal.c libsrc/Dk/Dkmarshal.c
index da15a21..b2b6c71 100644
--- a/libsrc/Dk/Dkmarshal.c
+++ b/libsrc/Dk/Dkmarshal.c
@@ -31,25 +31,7 @@
 # include <netinet/in.h>			 /* for ntohl, htonl */
 #endif
 
-#if defined (i386) || \
-    defined (_WIN64) || \
-    defined (_M_IX86) || \
-    defined (_M_ALPHA) || \
-    defined (mc68000) || \
-    defined (sparc) || \
-    defined (__x86_64) || \
-    defined (__alpha) || \
-    defined (__powerpc) || \
-    defined (mips) || \
-    defined (__OS2__) || \
-    defined (_IBMR2)
-# define _IEEE_FLOATS
-#elif defined (OPL_SOURCE)
-# include <librpc.h>
-#else
-# include <rpc/types.h>
-# include <rpc/xdr.h>
-#endif
+#define _IEEE_FLOATS
 
 macro_char_func readtable[256];
 ses_write_func writetable[256];
-- 
2.24.0

